docusaurus/docs/what-is-included/sitemap.md

```diff
@@ -0,0 +1,21 @@
+---
+id: sitemap-robots
+title: Sitemap & robots.txt
+sidebar_label: Sitemap & robots.txt
+---
+
+The boilerplate includes a dynamic sitemap generator and a robots.txt file. This will help search engines and their crawlers index your whole website, including static and dynamic URL's, giving you better results in SEO ranks. [`@moxy/next-sitemaps`](https://www.npmjs.com/package/@moxy/next-sitemaps) helps make this a very simple task. 
```

Comment:
```suggestion
The boilerplate includes a dynamic sitemap generator and a `robots.txt` file. This will help search engines and their crawlers index your whole website, including static and dynamic URL's, giving you better results in SEO ranks. [`@moxy/next-sitemaps`](https://www.npmjs.com/package/@moxy/next-sitemaps) helps make this a very simple task. 
```

Reply:
Can you also add link sitemap to a webpage describing what it is?